lib/opencpu/client.rb

```diff
@@ -107,7 +107,8 @@ def request_options(data, format)
     end
 
     def function_url(package, function, user = :system, github_remote = false, format = nil)
-      "#{package_url(package, user, github_remote)}/R/#{function}/#{format.to_s}"
+      "#{package_url(package, user, github_remote)}/#{function}/#{format.to_s}"
```

Comment:
Yes. I want to generalize the function_url.

For example

- GET https://cloud.opencpu.org/ocpu/library/MASS/data/Boston/json
- GET https://cloud.opencpu.org/ocpu/library/MASS/scripts/
- POST https://cloud.opencpu.org/ocpu/library/MASS/scripts/ch01.R

## :clipboard: Reference
1. https://www.opencpu.org/api.html

Reply:
Problem is that this will break everyones code. so the mayor version would have to be updated. Feels like the other calls might be better as different endpoints in client? since they are not a function calls and will also not usually work the same way? Can scripts return multiple resources? Also, prefixing all function calls with `R/` feels like what a wrapper is there for.